CMakeLists.txt

```diff
@@ -93,6 +93,15 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+set(_swift_install_apinotes_default FALSE)
+if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  set(_swift_install_apinotes_default TRUE)
+endif()
+
+option(SWIFT_INSTALL_APINOTES
```

Comment:
Maybe this should be called `SWIFT_INCLUDE_APINOTES`? It would be more in-line with other options like it. This does control if the targets are created, but not if they're actually installed. You still need to be installing the `compiler` component for them to be actually installed.